src/Console/ConfigurationResolver.php

```diff
@@ -563,6 +566,48 @@ private function parseRules()
     }
 
     /**
+     * @param array $rules
+     *
+     * @throws InvalidConfigurationException
+     */
+    private function validateRules(array $rules)
+    {
+        /**
+         * Create a ruleset that contains all configured rules, even when they originally have been disabled.
+         *
+         * @see RuleSet::resolveSet()
+         */
+        $ruleSet = RuleSet::create(array_map(function () {
+            return true;
+        }, $rules));
+
+        /* @var string[] $availableFixers */
+        $configuredFixers = array_keys($ruleSet->getRules());
+
+        $fixerFactory = new FixerFactory();
+        $fixerFactory->registerBuiltInFixers();
+        $fixerFactory->registerCustomFixers($this->getConfig()->getCustomFixers());
+
+        $reflection = new \ReflectionProperty('PhpCsFixer\FixerFactory', 'fixersByName');
+        $reflection->setAccessible(true);
```

Comment:
Hmm. The problem is, how do I find out which fixers have been registered by name? Should I add an accessor to `FixerFactory`, then?